lib/usd/utils/MergePrimsOptions.cpp

```diff
@@ -0,0 +1,191 @@
+        TF_CODING_ERROR(
+            "Dictionary is missing required key '%s' or key is "
+            "not a vector of tokens",
+            key.GetText());
+        return def;
+    }
+
+    MergeVerbosity verbosity = MergeVerbosity::None;
+
+    const auto tokens = VtDictionaryGet<std::vector<VtValue>>(options, key);
+    for (const auto& token : tokens) {
+        if (UsdMayaMergeOptionsTokens->None == token)
+            verbosity = verbosity | MergeVerbosity::None;
+        if (UsdMayaMergeOptionsTokens->Same == token)
+            verbosity = verbosity | MergeVerbosity::Same;
+        if (UsdMayaMergeOptionsTokens->Differ == token)
+            verbosity = verbosity | MergeVerbosity::Differ;
+        if (UsdMayaMergeOptionsTokens->Child == token)
+            verbosity = verbosity | MergeVerbosity::Child;
+        if (UsdMayaMergeOptionsTokens->Children == token)
+            verbosity = verbosity | MergeVerbosity::Children;
+        if (UsdMayaMergeOptionsTokens->Failure == token)
+            verbosity = verbosity | MergeVerbosity::Failure;
+        if (UsdMayaMergeOptionsTokens->Default == token)
+            verbosity = verbosity | MergeVerbosity::Default;
+    }
+
+    return verbosity;
+}
+
+/// Extracts a MergeMissing array of tokens at \p key from \p options, or All if it can't
+/// extract.
+MergeMissing parseMissingHandling(
+    const VtDictionary& options,
+    const TfToken&      key,
+    MergeMissing        def = MergeMissing::All)
+{
+    if (!VtDictionaryIsHolding<std::vector<VtValue>>(options, key)) {
+        TF_CODING_ERROR(
+            "Dictionary is missing required key '%s' or key is "
+            "not a vector of tokens",
+            key.GetText());
+        return def;
+    }
+
+    MergeMissing missingHandling = MergeMissing::None;
+
+    const auto tokens = VtDictionaryGet<std::vector<VtValue>>(options, key);
+    for (const auto& token : tokens) {
+        if (UsdMayaMergeOptionsTokens->None == token)
+            missingHandling = missingHandling | MergeMissing::None;
+        if (UsdMayaMergeOptionsTokens->Create == token)
+            missingHandling = missingHandling | MergeMissing::Create;
+        if (UsdMayaMergeOptionsTokens->Preserve == token)
+            missingHandling = missingHandling | MergeMissing::Preserve;
+        if (UsdMayaMergeOptionsTokens->All == token)
+            missingHandling = missingHandling | MergeMissing::All;
+    }
+
+    return missingHandling;
+}
+
+} // namespace
+
+/* static */
+const VtDictionary& MergePrimsOptions::getDefaultDictionary()
+{
+    static VtDictionary   d;
+    static std::once_flag once;
+    std::call_once(once, []() {
+        const auto _token = VtValue(TfToken());
+        const auto _tokenVector = VtValue(std::vector<VtValue>({ _token }));
+
+        d[UsdMayaMergeOptionsTokens->verbosity]
+            = VtValue(std::vector<VtValue>({ VtValue(UsdMayaMergeOptionsTokens->Default) }));
```

Comment:
I'm confused as to why this is a VtValue that's a vector of VtValue's.

Reply:
It's the list of verbosity value, which can be or'ed together. So it must support more than a single token.

Reply:
So it can be built incrementally by appending to the vector, and must be OR'ed at read time?  That seems nice and flexible, but a bit complicated.

Reply:
Well, I did not see a better way to express that is accept multiple tokens. The alternative would be to have all the possible combinations as tokens, but that creates 2 ^ N different tokens, which are all hard to read.

Reply:
If it's got to be expressed as tokens, then yes you're right.  Dumb question time: why isn't it a plain integer, rather than a bunch of tokens?

Reply:
Would be obscure for programmers. I also expected these to be exposed, either in Python on in MEL when doing UI for the options, having dict with integer would obscure the code.